### ISSUE TITLE
Multiple compendias have the same version

### ISSUE THREAD

**Issue by arielsvn** [AlexsLemonade/refinebio#1829]
### Context

On the endpoint `/v1/compendia` we use `compendia_version` to get the latest compendia version for a given organism.

### Problem or idea

Looks like we are not incrementing these values correctly:

```
    {
        "id": 5730851,
        "filename": "441b3c7a-ecf3-4f1f-966e-1e8509287ed9_compendia.zip",
        "size_in_bytes": 2735058985,
        "is_compendia": true,
        "quant_sf_only": false,
        "compendia_version": 1,
        "organism_name": "RATTUS_NORVEGICUS",
        "sha1": "803052c13ee666a365615bfd58450dae2f511083",
        "s3_bucket": "data-refinery-s3-circleci-prod",
        "s3_key": "0630wrqrrsjlfcpcw8a8lzaa_441b3c7a-ecf3-4f1f-966e-1e8509287ed9_compendia.zip",
        "s3_url": "https://s3.amazonaws.com/data-refinery-s3-circleci-prod/0630wrqrrsjlfcpcw8a8lzaa_441b3c7a-ecf3-4f1f-966e-1e8509287ed9_compendia.zip",
        "created_at": "2019-10-16T00:50:30.415087Z",
        "last_modified": "2019-10-16T00:52:56.537989Z"
    },
    {
        "id": 5730848,
        "filename": "aaaacc00-185d-441c-bd62-cf8ff07ff82f_compendia.zip",
        "size_in_bytes": 2735034298,
        "is_compendia": true,
        "quant_sf_only": false,
        "compendia_version": 1,
        "organism_name": "RATTUS_NORVEGICUS",
        "sha1": "56099d279531abf884c6f8786137a58a380b840d",
        "s3_bucket": "data-refinery-s3-circleci-prod",
        "s3_key": "gz82jtax6nz6wlwe1x68cth5_aaaacc00-185d-441c-bd62-cf8ff07ff82f_compendia.zip",
        "s3_url": "https://s3.amazonaws.com/data-refinery-s3-circleci-prod/gz82jtax6nz6wlwe1x68cth5_aaaacc00-185d-441c-bd62-cf8ff07ff82f_compendia.zip",
        "created_at": "2019-10-12T17:19:50.213335Z",
        "last_modified": "2019-10-12T17:22:11.032866Z"
    },
```

### Solution or next step

This is where we assign the `compendia_version`

https://github.com/AlexsLemonade/refinebio/blob/dev/workers/data_refinery_workers/processors/create_compendia.py#L438-L445

I'm wondering now if we could remove this value, and just use the date when the compendia was created to get the latest one.

**Comment by davidsmejia** [comment]
These should be the current latest versions based on what I can see is public. I am currently working on a migration to fix these. 

Normalized
``` → curl https://api.refine.bio/v1/compendia/\?quant_sf_only\=False\&ordering\=primary_organism__name\&limit\=1000\&format\=json | jq '.results[].primary_organism' | uniq -c
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 31769  100 31769    0     0  57867      0 --:--:-- --:--:-- --:--:-- 57867
   3 "BOS_INDICUS"
   1 "BOS_TAURUS"
   1 "CITRUS_SINENSIS"
   2 "DANIO_RERIO"
   1 "ESCHERICHIA_COLI"
   3 "ESCHERICHIA_COLI_K-12"
   1 "ESCHERICHIA_COLI_STR._K-12_SUBSTR._MG1655"
   1 "GALLUS_GALLUS"
   3 "GOSSYPIUM_HIRSUTUM"
   1 "HORDEUM_VULGARE"
   1 "LEPIDIUM_SATIVUM"
   1 "MACACA_FASCICULARIS"
   1 "MACACA_MULATTA"
   1 "MUSCULUS"
   1 "MUSTELA_PUTORIUS_FURO"
   1 "ORYZA_SATIVA_INDICA_GROUP"
   1 "PLASMODIUM_FALCIPARUM"
   1 "POPULUS_TREMULA_X_POPULUS_ALBA"
   1 "POPULUS_TRICHOCARPA"
   1 "POPULUS_X_CANADENSIS"
   1 "PSEUDOMONAS_AERUGINOSA"
   1 "PSEUDOMONAS_AERUGINOSA_PAO1"
  11 "RATTUS_NORVEGICUS"
   1 "SCHIZOSACCHAROMYCES_POMBE"
   1 "STAPHYLOCOCCUS_AUREUS"
   1 "SUS_SCROFA"
   1 "VITIS_HYBRID_CULTIVAR"
   1 "VITIS_RIPARIA"
   1 "VITIS_VINIFERA"
   2 "XENOPUS_LAEVIS"
   1 "ZEA_MAYS"
```

Quantile Normalized:
```   1 "ACTINIDIA_CHINENSIS"
   1 "ACYRTHOSIPHON_PISUM"
   1 "AEDES_AEGYPTI"
   1 "AEGILOPS_TAUSCHII"
   1 "AILUROPODA_MELANOLEUCA"
   1 "AMPHILOPHUS_CITRINELLUS"
   1 "AMPHIMEDON_QUEENSLANDICA"
   1 "AMPHIPRION_OCELLARIS"
   1 "AMPHIPRION_PERCULA"
   1 "ANOLIS_CAROLINENSIS"
   1 "ANOPHELES_DARLINGI"
   2 "ANOPHELES_GAMBIAE"
   1 "ANOPLOPHORA_GLABRIPENNIS"
   1 "APIS_MELLIFERA"
   1 "APTERYX_OWENII"
   1 "ARABIDOPSIS_HALLERI"
   1 "ARABIDOPSIS_LYRATA"
   2 "ARABIDOPSIS_THALIANA"
   1 "ASTATOTILAPIA_CALLIPTERA"
   1 "BETA_VULGARIS"
   1 "BOMBUS_TERRESTRIS"
   1 "BOMBYX_MORI"
   2 "BOS_TAURUS"
   1 "BRACHYPODIUM_DISTACHYON"
   1 "BRASSICA_NAPUS"
   1 "BRASSICA_OLERACEA"
   1 "BRASSICA_RAPA"
   1 "BRUGIA_MALAYI"
   1 "CAENORHABDITIS_BRENNERI"
   1 "CAENORHABDITIS_BRIGGSAE"
   2 "CAENORHABDITIS_ELEGANS"
   1 "CAENORHABDITIS_JAPONICA"
   1 "CAENORHABDITIS_REMANEI"
   1 "CALLITHRIX_JACCHUS"
   1 "CAPITELLA_TELETA"
   1 "CAPRA_HIRCUS"
   1 "CAVIA_APEREA"
   1 "CAVIA_PORCELLUS"
   1 "CERCOCEBUS_ATYS"
   1 "CHINCHILLA_LANIGERA"
   1 "CHLAMYDOMONAS_REINHARDTII"
   1 "CHLOROCEBUS_SABAEUS"
   1 "CIONA_INTESTINALIS"
   1 "CIONA_SAVIGNYI"
   1 "CITRUS_SINENSIS"
   1 "CORCHORUS_CAPSULARIS"
   1 "COTURNIX_JAPONICA"
   1 "CRASSOSTREA_GIGAS"
   1 "CROCODYLUS_POROSUS"
   1 "CUCUMIS_SATIVUS"
   1 "CULEX_QUINQUEFASCIATUS"
   1 "CULICOIDES_SONORENSIS"
   1 "CYPRINODON_VARIEGATUS"
   3 "DANIO_RERIO"
   1 "DAPHNIA_MAGNA"
   1 "DAPHNIA_PULEX"
   1 "DASYPUS_NOVEMCINCTUS"
   1 "DAUCUS_CAROTA"
   1 "DENDROCTONUS_PONDEROSAE"
   1 "DROMAIUS_NOVAEHOLLANDIAE"
   1 "DROSOPHILA_ANANASSAE"
   1 "DROSOPHILA_ERECTA"
   1 "DROSOPHILA_GRIMSHAWI"
   2 "DROSOPHILA_MELANOGASTER"
   1 "DROSOPHILA_MOJAVENSIS"
   1 "DROSOPHILA_PERSIMILIS"
   1 "DROSOPHILA_PSEUDOOBSCURA"
   1 "DROSOPHILA_SECHELLIA"
   1 "DROSOPHILA_SIMULANS"
   1 "DROSOPHILA_VIRILIS"
   1 "DROSOPHILA_WILLISTONI"
   1 "DROSOPHILA_YAKUBA"
   1 "EPTATRETUS_BURGERI"
   2 "EQUUS_CABALLUS"
   1 "ESCHERICHIA_COLI"
   1 "ESOX_LUCIUS"
   1 "FELIS_CATUS"
   1 "FOLSOMIA_CANDIDA"
   1 "FUNDULUS_HETEROCLITUS"
   1 "GADUS_MORHUA"
   1 "GALDIERIA_SULPHURARIA"
   2 "GALLUS_GALLUS"
   1 "GAMBUSIA_AFFINIS"
   1 "GASTEROSTEUS_ACULEATUS"
   2 "GLYCINE_MAX"
   1 "GOSSYPIUM_HIRSUTUM"
   1 "GOSSYPIUM_RAIMONDII"
   1 "HELIANTHUS_ANNUUS"
   1 "HELICONIUS_MELPOMENE"
   1 "HOMO_SAPIENS"
   1 "HORDEUM_VULGARE"
   1 "ICTIDOMYS_TRIDECEMLINEATUS"
   1 "IXODES_SCAPULARIS"
   1 "JUNCO_HYEMALIS"
   1 "LATIMERIA_CHALUMNAE"
   1 "LEPEOPHTHEIRUS_SALMONIS"
   1 "LEPIDIUM_SATIVUM"
   1 "LEPISOSTEUS_OCULATUS"
   1 "LOA_LOA"
   1 "LOXODONTA_AFRICANA"
   1 "LUCILIA_CUPRINA"
   1 "LUPINUS_ANGUSTIFOLIUS"
   1 "MACACA_MULATTA"
   1 "MACACA_NEMESTRINA"
   1 "MANACUS_VITELLINUS"
   1 "MANIHOT_ESCULENTA"
   1 "MASTACEMBELUS_ARMATUS"
   1 "MAYETIOLA_DESTRUCTOR"
   1 "MAYLANDIA_ZEBRA"
   1 "MEDICAGO_TRUNCATULA"
   1 "MELEAGRIS_GALLOPAVO"
   1 "MELITAEA_CINXIA"
   1 "MICROTUS_OCHROGASTER"
   1 "MNEMIOPSIS_LEIDYI"
   1 "MONODELPHIS_DOMESTICA"
   1 "MUSA_ACUMINATA"
   1 "MUS_CAROLI"
   1 "MUS_MUSCULUS"
   1 "MUS_PAHARI"
   1 "MUS_SPRETUS"
   2 "MUSTELA_PUTORIUS_FURO"
   1 "MYOTIS_LUCIFUGUS"
   1 "NASONIA_VITRIPENNIS"
   1 "NEMATOSTELLA_VECTENSIS"
   1 "NEOLAMPROLOGUS_BRICHARDI"
   1 "NEOVISON_VISON"
   1 "NICOTIANA_ATTENUATA"
   1 "NOTAMACROPUS_EUGENII"
   1 "NOTECHIS_SCUTATUS"
   1 "NOTHOPROCTA_PERDICARIA"
   1 "NUMIDA_MELEAGRIS"
   1 "OCTOPUS_BIMACULOIDES"
   1 "ONCHOCERCA_VOLVULUS"
   1 "OREOCHROMIS_NILOTICUS"
   1 "ORYCTOLAGUS_CUNICULUS"
   1 "ORYZA_GLABERRIMA"
   1 "ORYZA_LONGISTAMINATA"
   1 "ORYZA_RUFIPOGON"
   2 "ORYZA_SATIVA"
   1 "ORYZIAS_LATIPES"
   1 "ORYZIAS_MELASTIGMA"
   1 "OTOLEMUR_GARNETTII"
   2 "OVIS_ARIES"
   1 "PANTHERA_TIGRIS_ALTAICA"
   1 "PARUS_MAJOR"
   1 "PELODISCUS_SINENSIS"
   1 "PETROMYZON_MARINUS"
   1 "PHASCOLARCTOS_CINEREUS"
   1 "PHASEOLUS_VULGARIS"
   1 "PHYSCOMITRELLA_PATENS"
   1 "POECILIA_MEXICANA"
   1 "PONGO_ABELII"
   2 "POPULUS_TRICHOCARPA"
   1 "PRISTIONCHUS_PACIFICUS"
   1 "PRUNUS_PERSICA"
   1 "PSEUDOMONAS_AERUGINOSA"
   1 "PUNDAMILIA_NYEREREI"
   2 "RATTUS_NORVEGICUS"
   1 "RHODNIUS_PROLIXUS"
   2 "SACCHAROMYCES_CEREVISIAE"
   1 "SARCOPHILUS_HARRISII"
   1 "SCHISTOSOMA_MANSONI"
   1 "SCHIZOSACCHAROMYCES_POMBE"
   1 "SCLEROPAGES_FORMOSUS"
   1 "SCOPHTHALMUS_MAXIMUS"
   1 "SELAGINELLA_MOELLENDORFFII"
   1 "SETARIA_ITALICA"
   1 "SOLANUM_TUBEROSUM"
   1 "SOLENOPSIS_INVICTA"
   1 "SORGHUM_BICOLOR"
   1 "SPHENODON_PUNCTATUS"
   1 "STAPHYLOCOCCUS_AUREUS"
   1 "STRIGAMIA_MARITIMA"
   1 "STRONGYLOCENTROTUS_PURPURATUS"
   1 "STRONGYLOIDES_RATTI"
   2 "SUS_SCROFA"
   1 "TAENIOPYGIA_GUTTATA"
   1 "TELEOPSIS_DALMANNI"
   1 "TETRANYCHUS_URTICAE"
   1 "TETRAODON_NIGROVIRIDIS"
   1 "THEOBROMA_CACAO"
   1 "TRIBOLIUM_CASTANEUM"
   1 "TRICHINELLA_SPIRALIS"
   1 "TRICHOPLAX_ADHAERENS"
   1 "TRIFOLIUM_PRATENSE"
   1 "TRITICUM_AESTIVUM"
   1 "TRITICUM_URARTU"
   1 "TUPAIA_BELANGERI"
   1 "TURSIOPS_TRUNCATUS"
   1 "VIGNA_ANGULARIS"
   1 "VIGNA_RADIATA"
   2 "VITIS_VINIFERA"
   1 "VULPES_VULPES"
   1 "XENOPUS_LAEVIS"
   1 "XENOPUS_TROPICALIS"
   1 "XIPHOPHORUS_COUCHIANUS"
   1 "XIPHOPHORUS_MACULATUS"
   2 "ZEA_MAYS"
   1 "ZONOTRICHIA_ALBICOLLIS"
   1 "ZOOTERMOPSIS_NEVADENSIS"
```

**Comment by arielsvn** [comment]
fixed by #1939
